plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -910,6 +912,255 @@ public void testSettingChangeDataFeedEnabledProperty()
+        assertUpdate("INSERT INTO " + tableName + " VALUES('url4', 'domain4', 4), ('url5', 'domain5', 2), ('url6', 'domain6', 6)", 3);
+
+        assertUpdate("UPDATE " + tableName + " SET page_url = 'url22' WHERE views = 2", 2);
+        MaterializedResult result = computeActual("SELECT * FROM TABLE(system.table_changes('test_schema', '" + tableName + "'))");
+        List<MaterializedRow> resultWithoutTimestamp = getCdfColumnsWithoutTimestamp(result);
+        assertThat(resultWithoutTimestamp).containsOnly(
+                new MaterializedRow(ImmutableList.of("url1", "domain1", 1, "insert", 1)),
+                new MaterializedRow(ImmutableList.of("url2", "domain2", 2, "insert", 1)),
+                new MaterializedRow(ImmutableList.of("url3", "domain3", 3, "insert", 1)),
+                new MaterializedRow(ImmutableList.of("url4", "domain4", 4, "insert", 2)),
+                new MaterializedRow(ImmutableList.of("url5", "domain5", 2, "insert", 2)),
+                new MaterializedRow(ImmutableList.of("url6", "domain6", 6, "insert", 2)),
+                new MaterializedRow(ImmutableList.of("url2", "domain2", 2, "update_preimage", 3)),
+                new MaterializedRow(ImmutableList.of("url22", "domain2", 2, "update_postimage", 3)),
+                new MaterializedRow(ImmutableList.of("url5", "domain5", 2, "update_preimage", 3)),
+                new MaterializedRow(ImmutableList.of("url22", "domain5", 2, "update_postimage", 3)));
+
+        assertUpdate("DELETE FROM " + tableName + " WHERE views = 2", 2);
+        result = computeActual("SELECT * FROM TABLE(system.table_changes('test_schema', '" + tableName + "', 3))");
+        resultWithoutTimestamp = getCdfColumnsWithoutTimestamp(result);
+        assertThat(resultWithoutTimestamp).containsOnly(
+                new MaterializedRow(ImmutableList.of("url22", "domain2", 2, "delete", 4)),
+                new MaterializedRow(ImmutableList.of("url22", "domain5", 2, "delete", 4)));
+    }
+
+    @Test
+    public void testReadingChangesMadeByBasicOperationsOnPartitionedTableWithCdfEnabled()
+    {
+        String tableName = "test_basic_operations_on_table_with_cdf_enabled_" + randomNameSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " (page_url VARCHAR, domain VARCHAR, views INTEGER) WITH (change_data_feed_enabled = true, partitioned_by = ARRAY['domain'])");
+        assertUpdate("INSERT INTO " + tableName + " VALUES('url1', 'domain1', 1), ('url2', 'domain2', 2), ('url3', 'domain1', 3)", 3);
+        assertUpdate("INSERT INTO " + tableName + " VALUES('url4', 'domain1', 400), ('url5', 'domain2', 500), ('url6', 'domain3', 2)", 3);
+
+        assertUpdate("UPDATE " + tableName + " SET domain = 'domain4' WHERE views = 2", 2);
+        assertQuery("SELECT * FROM " + tableName, "VALUES('url1', 'domain1', 1), ('url2', 'domain4', 2), ('url3', 'domain1', 3)," +
```

Comment:
Add the query and expected result on separate lines.
`VALUES ...` in a multiline text block